tests/all/js_globals/Number.rs

```diff
@@ -3,7 +3,9 @@
 use super::project;
 
 
-#[test]
+// TODO: test seems to be system dependent, disable for now
+// #[test]
+#[allow(dead_code)]
```

Comment:
Delete these two lines and it'll be fine. 
https://github.com/rustwasm/wasm-bindgen/pull/302/files#diff-c29ca216e01d6480124fdf536ba30e97R29
https://github.com/rustwasm/wasm-bindgen/pull/302/files#diff-c29ca216e01d6480124fdf536ba30e97R31

Reply:
Good point.